test/regression.uts

```diff
@@ -1639,7 +1639,8 @@ retry_test(_test)
 def _test():
     ret = conf.AS_resolver.resolve("8.8.8.8", "8.8.4.4")
     assert (len(ret) == 2)
-    all(x[1] == "AS15169" for x in ret)
+    all(x[1] in {15169, 4205777769} for x in ret)
```

Comment:
FTR, this is the private AS 32bits range. It it likely a local BGP announcements that leaks on the Internet.